src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -379,6 +379,107 @@ class AppsResourceTest extends AkkaUnitTest with GroupCreation with JerseyTest {
       }
     }
 
+    "Accept an app definition with ipcMode defined private and shmSize set" in new Fixture {
+      Given("an app with ipcMode defined private and shmSize set")
+      val container = RamlContainer(
+        `type` = EngineType.Mesos,
+        docker = Option(DockerContainer(image = "private/image")),
```

Comment:
Sort of an unwritten convention is to use `Some(value)` instead of `Option(value)` here (and below)

Reply:
I've changed it to Some in the whole file